package.json

```diff
@@ -96,7 +96,7 @@
     "@types/mocha": "^7.0.2",
     "@types/node": "^13.11.0",
     "@types/text-encoding": "^0.0.35",
-    "@types/vscode": "^1.46.0",
+    "@types/vscode": "^1.67.0",
```

Comment:
Is it required to update the version? because this could possibly be breaking otherwise.